src/core/Akka.Persistence.Tests/AtLeastOnceDeliverySpec.cs

```diff
@@ -15,7 +15,6 @@
 using Xunit;
 using Xunit.Abstractions;
 using FluentAssertions;
-#pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
```

Comment:
Got rid of the warning and fixed the problems instead. These are all related to the test messages themselves so no real impact on Akka.NET.